oojs.js

```diff
@@ -4,9 +4,32 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
+  class Playlist {
+    constructor(title, song, favorites) {
+      this.title = title;
+      this.song = song;
```

Comment:
`this.song` should be plural, and then your constructor parameter `song` would be assigned like this:
```js
this.songs = [];
if(song) this.songs.push(song);
```